### PR TITLE
admin: some tweaks

### PR DESCRIPTION
**Pull request by iDannz1**
This PR replaces deprecated functions, removes unused code, fixes calls to non-existing functions, etc...

I found them using [this](https://github.com/sumneko/vscode-lua), and maybe in the near future, i can send other PRs doing the same to other resources.

### CODE REVIEW

File: [admin]/admin/client/gui/admin_vehicle.lua [hunk 1]
```diff
@@ -241,9 +241,9 @@ function aClientVehicleClick ( button )
 			triggerServerEvent ( "aVehicle", getLocalPlayer(), aVehicleCustomizePlayer, "setlights", { guiGetText ( aLightsColor ) } )
 		elseif ( source == aLicensePlateSet ) then
 			triggerServerEvent ( "aVehicle", getLocalPlayer(), aVehicleCustomizePlayer, "setplates", { guiGetText ( aLicensePlate ) } )
-		elseif ( source == aVehicleColorScheme ) then
-			guiSetVisible ( aVehicleColorForm, true )
-			guiBringToFront ( aVehicleColorForm )
```
Comment (iDannz1) [inline [admin]/admin/client/gui/admin_vehicle.lua:246]: this GUI element was disabled some time ago
https://github.com/multitheftauto/mtasa-resources/blob/44053e2c7b59b84183cf1a349822c853354aa76f/%5Badmin%5D/admin/client/gui/admin_vehicle.lua#L77

File: [admin]/admin/server/admin_sync.lua [hunk 2]
```diff
@@ -122,20 +122,6 @@ function aSynchCoroutineFunc( type, data, typeOfTag, banSearchTag )
 		tableOut["map"] = getMapName()
 		tableOut["password"] = getServerPassword()
 		tableOut["fps"] = getFPSLimit()
-	elseif ( type == "rights" ) then
```
Comment (iDannz1) [inline [admin]/admin/server/admin_sync.lua:127]: This part is never executed, and even if it was, the code inside it doesn't make sense.

File: [admin]/admin/server/admin_sync.lua [hunk 3]
```diff
@@ -122,20 +122,6 @@ function aSynchCoroutineFunc( type, data, typeOfTag, banSearchTag )
 		tableOut["map"] = getMapName()
 		tableOut["password"] = getServerPassword()
 		tableOut["fps"] = getFPSLimit()
-	elseif ( type == "rights" ) then
-		for gi, group in ipairs ( aclListGroups() ) do
```
Comment (iDannz1) [inline [admin]/admin/server/admin_sync.lua:128]: `aclListGroups` is not defined anywhere and it's not a default function

File: [admin]/admin/server/admin_sync.lua [hunk 4]
```diff
@@ -122,20 +122,6 @@ function aSynchCoroutineFunc( type, data, typeOfTag, banSearchTag )
 		tableOut["map"] = getMapName()
 		tableOut["password"] = getServerPassword()
 		tableOut["fps"] = getFPSLimit()
-	elseif ( type == "rights" ) then
-		for gi, group in ipairs ( aclListGroups() ) do
-			for oi, object in ipairs ( aclGroupListObjects ( group ) ) do
-				if ( ( object == data ) or ( object == "user.*" ) ) then
-					for ai, acl in ipairs ( aclGroupListACL ( group ) ) do
-						for ri, right in ipairs ( aclListRights ( acl ) ) do
-							local access = aclGetRight ( acl, string )
```
Comment (iDannz1) [inline [admin]/admin/server/admin_sync.lua:133]: `string` is not defined anywhere

File: [admin]/admin/server/admin_sync.lua [hunk 5]
```diff
@@ -122,20 +122,6 @@ function aSynchCoroutineFunc( type, data, typeOfTag, banSearchTag )
...
 		tableOut["password"] = getServerPassword()
 		tableOut["fps"] = getFPSLimit()
-	elseif ( type == "rights" ) then
-		for gi, group in ipairs ( aclListGroups() ) do
-			for oi, object in ipairs ( aclGroupListObjects ( group ) ) do
-				if ( ( object == data ) or ( object == "user.*" ) ) then
-					for ai, acl in ipairs ( aclGroupListACL ( group ) ) do
-						for ri, right in ipairs ( aclListRights ( acl ) ) do
-							local access = aclGetRight ( acl, string )
-							if ( access ) then table.insert ( tableOut, right ) end
```
Comment (iDannz1) [inline [admin]/admin/server/admin_sync.lua:134]: never executed, because `access` will always be false

**Review by jlillis — APPROVED** [review]
This looks good after a brief review. Nice work.

**Comment by ricksterhd123** [comment]
Did you have to setup vscode-lua in some way to stop warning about mta functions? I remember trying similar tools and it wouldn't stop complaining about mta functions. I have a lot of old code to refactor and this tool would be great. Perhaps you could fork it and add MTA specific features such as spotting deprecated functions etc? I can imagine it would be very popular among scripters.

**Comment by ricksterhd123** [comment]
On the same topic: Dutch showed me this before: https://github.com/Koihik/LuaFormatter and it looks promising perhaps we could configure a standard formating for mtasa resources. Only issues I had with it was that lambda functions would look weird but i think it's configurable. Shall we open a discussion? Maybe it's unecessary but I think it would be worth it.

**Comment by iDannz1** [comment]
> On the same topic: Dutch showed me this before: https://github.com/Koihik/LuaFormatter and it looks promising perhaps we could configure a standard formating for mtasa resources. Only issues I had with it was that lambda functions would look weird but i think it's configurable. Shall we open a discussion? Maybe it's unecessary but I think it would be worth it.

Maybe we could use <https://github.com/multitheftauto/mtasa-resources/issues/250> to discuss this subject

**Comment by Dutchman101** [comment]
Tested on a high traffic server for over a month, and confirmed to be stable.

**Pull request merged**